conf/interestRates.csv

```diff
@@ -1,4 +1,5 @@
-19 Mar 2020,2.60
```

Comment:
https://www.gov.uk/government/publications/rates-and-allowances-hmrc-interest-rates-for-late-and-early-payments/rates-and-allowances-hmrc-interest-rates